packages/SidePanel/src/components/Dialog/Dialog.js

```diff
@@ -77,6 +81,7 @@ function Dialog(props) {
   return (
     <sc.Dialog
       aria-modal={isInline ? null : "true"}
+      getPushContentElement={getPushContentElement}
```

Comment:
maybe rename to `hasPushedElement={!!getPushContentElement}` ? since it's only used in styling part